### DISCUSSION TITLE
Cant connect

### DISCUSSION THREAD

**Discussion by mkultrahh** [mattwebbio/orbital-sync#134] [General]
Hi

I seem to be having problems connecting to the second pi-hole in my setup. The second pi-hole is run through docker on unraid. 

seem to be getting EHOSTUNREACH even though when I click on the link it works? triple checked password. 

Any advice?  

![Screenshot 2023-01-30 161002](https://user-images.githubusercontent.com/100877286/215530744-42d2b3b3-8873-457a-a3c0-5bae3af93ef8.png)

**Comment by jacobslusser** [comment]
Is Orbital Sync running in the same docker engine as your Pi-hole instance? If so, you will likely need to use the internal docker assigned IP address (and container port) to Pi-hole, not the public IP and port. Docker containers communicate with each other over an internal network.

**Reply by Roving-Ronin** [reply]
Would anyone have resolved this? Runnning into the same problem (latest 2025 builds etc) with Primary PiHole on a RPi4 on one subnet and Secondary PiHole as a dock on UNRAID that has a dedicted bridge for VM/Dockers and its own static IP assigned to it.

**Comment by Cap10Canuck** [comment]
Same issue here.  I have two PiHoles running on RPi's, one of which is what I deem my "Master".  I have a third PiHole running on the same server that hosts the container for Orbital.  Orbital Sync can connect to both RPi's, but _cannot_ connect to the PiHole on the same machine it is on.

**Reply by Roving-Ronin** [reply]
You end up having to assign the docker a static IP for Orbital Sync to run. i.e. Change docker to network type is 'Custom' and use a NIC or Bridge (BR) of 802.2ad NIC's bundled together, and assign a static IP to the docker.

I think I might have also ended up also adding a bunch of extra variables to make sure Groups / Sync Lists / White and Black Lists etc also sync'ed.

![image](https://github.com/user-attachments/assets/47adad30-06ae-4c24-9930-1bb1412785b7)
